docs/api/example-0.0.2.tar.gz

```diff
@@ -0,0 +1,27 @@
+0 docs/
```

Comment:
I like the idea of this file but wondering about the `.tar.gz` part. Any good alternative? Something like `.tar.gz.sum` (also not too good).

Reply:
Renamed to `example-0.0.2.tar.gz-preview.txt`